packages/react-native-payments/lib/ios/ReactNativePayments.m

```diff
@@ -125,7 +126,7 @@ -(void) paymentAuthorizationViewControllerDidFinish:(PKPaymentAuthorizationViewC
     
     if (self.shippingContactCompletion) {
         // Display shipping address error when shipping is needed and shipping method count is below 1
-        if (self.initialOptions[@"requestShipping"] && [shippingMethods count] == 0) {
+        if (self.initialOptions[@"requestShipping"] && [shippingMethods count] == 0 || [invalidShipping isEqual:@(YES)]) {
```

Comment:
@crherman7 @andiskim 
Bools in dictionary are encapsulated in NSNumber objects. With that said can you change 
if (self.initialOptions[@"requestShipping"] && [shippingMethods count] == 0 || [invalidShipping isEqual:@(YES)]) to: 
if (self.initialOptions[@"requestShipping"] && [shippingMethods count] == 0 || [invalidShipping boolValue])

Reply:
actually since invalidShipping is not used outside of this, you can simply do: 
[details[@"invalidShipping"] boolValue]

Reply:
Fixed!